NuGet.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--<clear />-->
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
```

Comment:
This is for the preview testing packages. They're included in the default VS analyzer template - it's kinda crazy they're not on nuget though yeah. Frankly I currently have 0 idea how to get the CI to pass, but once I do, yeah, I'll look into this 😉 

```xml
<PackageReference Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.MSTest" Version="1.0.1-beta1.*" />
    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.MSTest" Version="1.0.1-beta1.*" />
    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeRefactoring.Testing.MSTest" Version="1.0.1-beta1.*" />
    <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Analyzer.Testing.MSTest" Version="1.0.1-beta1.*" />
    <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.CodeFix.Testing.MSTest" Version="1.0.1-beta1.*" />
    <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.CodeRefactoring.Testing.MSTest" Version="1.0.1-beta1.*" />
```

Reply:
FWIW, that feed is relied on by the dotnet sdk too. I guess it's their "dev" feed: https://github.com/dotnet/sdk/blob/main/NuGet.config

Reply:
It took me sufficient time to get this working in CI, that they've since released the packages to nuget.org, so this is no longer necessary. Removed in https://github.com/DataDog/dd-trace-dotnet/pull/1325/commits/a44bf18da36dc17bed5646816c135c49b518f9f5